### PR TITLE
#14 Javascript repository set up

### LINKED ISSUE #14: Repository set up - Javascript
This task is about creating a basic structure for the repository including Readme

### PR DESCRIPTION
**Pull request by rkokhatskyi**
### Description

This Pull Request closes #14 and creates structure of Javascript code including package.json and lock files. Readme updated accordingly

### Related Issue

Closes #14 

### Changes Made

1. Updated Readme to include more details
2. Added package.json
3. Installed dependencies and added lock files for both npm and yarn
4. Updated .gitignore to include additional unnecessary files

### By submitting this pull request, I confirm the following (please tick checkboxes):

- [x] Any new functions, methods, or classes are properly documented.
- [x] The commit messages are clear and descriptive.
- [x] The pull request title is concise and descriptive.
- [x] The pull request is targeted to the correct branch.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    {
+      "name": "Miguel Santamaria",
+      "github": "rkokhatskyi"
+    },
+    {
+      "name": "Roman Kokhatskyi",
+      "github": "rkokhatskyi"
+    }
+  ],
```
Comment (Agn21) [inline package.json:29]: Not that cool leaving me out!

"name" : "Angelos Nistikoulis",
"github": "Agn21"

**Review by WaltervonGabain — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      "name": "Jean-Christophe Rigo",
+      "github": "jcrigo86"
+    },
+    {
+      "name": "Walter von Gabain",
+      "github": "waltervongabain"
+    },
+    {
+      "name": "Miguel Santamaria",
+      "github": "rkokhatskyi"
```
Comment (msantamaria86) [inline package.json]: "github": "msantamaria86"

**Review by msantamaria86 — COMMENTED** [review]
2 minor corrections on contributors

**Comment by rkokhatskyi** [comment]
> 2 minor corrections on contributors

Updated @msantamaria86 @Agn21, thanks for the comments!

**Pull request merged**